src/networkcreator.js

```diff
@@ -1117,6 +1117,18 @@ class NetworkCreator extends EventEmitter {
       ],
       restart: 'unless-stopped'
     }
+
+    if (this.hasGeth) {
+      if (this.config.blockchain.minGasPrice) {
+        txFillerService.command.push('-minGasPrice')
+        txFillerService.command.push(this.config.blockchain.minGasPrice)
+      }
+      if (this.config.blockchain.maxGasPrice) {
+        txFillerService.command.push('-maxGasPrice')
+        txFillerService.command.push(this.config.blockchain.maxGasPrice)
+      }
+    }
+
```

Comment:
This works ! Just some nits here and there that could make it a little bit shorter. 

We could omit the `this.hasGeth` check since that's being checked in the return statement's ternary expression.

To then check whether an object key exists in an object we won't know will be defined we can do `(object.foo||{}).bar` and not run into `cannot access field bar of undefined`  

Javascript's `Array.push` just like go's append can take multiple arguments to append to the array so we could shorten this to: 

```
    if ((this.config.blockchain||{}).minGasPrice) {
      txFillerService.command.push('-minGasPrice', this.config.blockchain.minGasPrice)
    }
    if ((this.config.blockchain||{}).maxGasPrice) {
      txFillerService.command.push('-maxGasPrice', this.config.blockchain.maxGasPrice)
    }
```

Reply:
Thanks for the tip!

Fixed in 82311d4